e2e/app.spec.ts

```diff
@@ -148,21 +168,81 @@ describe('App', function() {
     actionListItemsFinder = await actionsListPage.getActionListItems();
     expect(actionListItemsFinder).to.have.lengthOf(6);
 
+    /**
+     * 6. Clicking on the first action on the actions list page, and thus displaying the correct Action page.
+     */
+
     // Click on the first action item
-    actionListItemsFinder[0].click();
+    actionListItemsFinder[ 0 ].click();
 
     // Wait for the Action page to show up on the DOM
     const actionPageFinder = actionPage.getPage();
     await presenceOf(actionPageFinder);
-    await expectDisplayed(actionPageFinder, 'Action Page is not displayed while it should be.');
+    await expectDisplayed(actionPageFinder, { elementName: 'Action Page' });
 
     // Ensure that the navbar title is indeed the expected title.
     const actionPageTitleFinder = await actionPage.getPageTitle();
     await expect(actionPageTitleFinder.getText()).to.eventually.have.string(actionPage.expectedTitle);
 
     // Ensure that the content of the page matches the action data
     const actionDetailsText = await actionPage.getActionDetails().getText();
-    expect(actionDetailsText).to.have.string(actions[0].get('title'));
-    expect(actionDetailsText).to.have.string(actions[0].get('description'));
+    expect(actionDetailsText).to.have.string(actions[ 0 ].get('title'));
+    expect(actionDetailsText).to.have.string(actions[ 0 ].get('description'));
+
+    /**
+     * 7. Clicking on the theme on the action page, and thus displaying the correct Theme page.
+     */
+
+    // Click on the theme
+    const actionPageThemeTitleFinder = await actionPage.getThemeTitle();
+    await expect(actionPageThemeTitleFinder.isPresent()).to.eventually.equal(true);
+    await browser.wait(EC.elementToBeClickable(actionPageThemeTitleFinder), 5000);
+
+    // Click on the theme title
+    actionPageThemeTitleFinder.click();
```

Comment:
Similarly to my comment in a previous PR, this should be encapsulated into a `goToTheme` method in `ActionPageObject`.